### DISCUSSION TITLE
network/virtual-nic

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#201] [General]
# network/virtual-nic

构建大规模高可用的分布式系统

https://thebyte.com.cn/network/virtual-nic.html

**Comment by xxh160** [comment]
“tun/tap 设备通常用作 overlay 网络传输，如图示例，应用程序通过 tun 发送数据包，tun 设备如果发现另一读被 VPN 程序打开” 有一个错别字 “另一<读>”

**Reply by isno** [reply]
谢谢，已修正。

**Comment by ZDWWWWW** [comment]
【Veth 就是带着两个“水晶头”的一根“网线” 】 好妙的描述

**Comment by zjyyzhengli** [comment]
将关联的 Veth 设备**其移动**到其他网络命名空间后 
-- 加粗的文字阅读时不是很通顺，"其"是否多余？

**Reply by isno** [reply]
是多余了，我改改。
